components/expenses/PayExpenseModal.js

```diff
@@ -213,20 +235,25 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, host, error,
   const hasAutomaticManualPicker = ![PayoutMethodType.OTHER, PayoutMethodType.ACCOUNT_BALANCE].includes(
     payoutMethodType,
   );
-  const canQuote =
-    host.transferwise && payoutMethodType === PayoutMethodType.BANK_ACCOUNT && formik.values.feesPayer !== 'PAYEE';
+  const canQuote = host.transferwise && payoutMethodType === PayoutMethodType.BANK_ACCOUNT;
   const quoteQuery = useQuery(quoteExpenseQuery, {
     variables: { id: expense.id },
     context: API_V2_CONTEXT,
     skip: !canQuote,
   });
 
-  const paymentProcessorFee = getPaymentProcessorFee(formik, expense, quoteQuery);
-  const isMultiCurrency =
-    expense?.amountInAccountCurrency && expense.amountInAccountCurrency.currency !== expense.currency;
-  const totalAmount = getTotalPayoutAmount(expense, { paymentProcessorFee, feesPayer: formik.values.feesPayer });
+  const paymentProcessorFee = getPaymentProcessorFee(formik, expense, quoteQuery, host);
```

Comment:
Looks like this is now used only to know whether there is a payment processor fee. I suggest:
- Getting rid of `getPaymentProcessorFee` (moreover, I believe the ways fees are computed there might be incorrect)
- Replacing the few cases we have of using this by `Boolean(amount.paymentProcessorFee)`

Reply:
Lol, obviously! Thanks!